apps/src/lib/tools/jsdebugger/JsDebugger.story.jsx

```diff
@@ -2,62 +2,99 @@ import React from 'react';
 import {Provider} from 'react-redux';
```

Comment:
The JsDebugger story appears to have broken pretty badly over the last few months; I'm not sure which change caused it, but having it default to closed may be part of the cause, and another issue is that the UnconnectedJsDebugger was using connected DebuggerButtons, so we need to provide a store in every story here.  In any case, the existing stories weren't making a lot of sense anymore so I pretty much rewrote this.